SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/LegacyLoadbalancingComponents.h

```diff
@@ -182,4 +182,86 @@ struct LegacyLB_VirtualWorkerAssignment
 
 	uint32 Virtual_worker_id;
 };
+
+struct LegacyLB_CustomWorkerAssignments
+{
+	static constexpr Worker_ComponentId ComponentId = 180500;
+
+	LegacyLB_CustomWorkerAssignments() {}
+
+	LegacyLB_CustomWorkerAssignments(const ComponentData& Data)
+		: LegacyLB_CustomWorkerAssignments(Data.GetUnderlying())
+	{
+	}
+
+	LegacyLB_CustomWorkerAssignments(Schema_ComponentData* Data)
+	{
+		Schema_Object* ComponentObject = Schema_GetComponentDataFields(Data);
+		ReadFromObject(ComponentObject);
+	}
+
+	ComponentData CreateComponentData() const
+	{
+		ComponentData Data(ComponentId);
+		Schema_Object* ComponentObject = Schema_GetComponentDataFields(Data.GetUnderlying());
+
+		WriteToObject(ComponentObject);
+
+		return Data;
+	}
+
+	ComponentUpdate CreateComponentUpdate() const
+	{
+		ComponentUpdate Update(ComponentId);
+		Schema_Object* ComponentObject = Schema_GetComponentUpdateFields(Update.GetUnderlying());
+
+		WriteToObject(ComponentObject);
+
+		return Update;
+	}
+
+	void ApplyComponentUpdate(const ComponentUpdate& Update) { ApplyComponentUpdate(Update.GetUnderlying()); }
+
+	void ApplyComponentUpdate(Schema_ComponentUpdate* Update)
+	{
+		Schema_Object* ComponentObject = Schema_GetComponentUpdateFields(Update);
+		ReadFromObject(ComponentObject);
+	}
+
+	void ReadFromObject(Schema_Object* Object)
+	{
+		LabelToVirtualWorker.Empty();
+		AdditionalInterest.Empty();
+		uint32 NumEntries = Schema_GetUint32Count(Object, 1);
```

Comment:
Where is the structure of this schema defined? E.g. what defines what id is tied to what entry.

Reply:
Right, forgot to add that : https://github.com/spatialos/UnrealGDKTestGyms/pull/463

Reply:
Can we have a comment that references this file location and repo.

Reply:
Done.